src/includes/platforms/configuration/integrations/plugin.mdx

```diff
@@ -34,7 +34,13 @@ Available options:
 
 _Import name: `Sentry.Integrations.CaptureConsole`_
 
-This integration captures all `Console API` calls and redirects them to Sentry using the SDK's `captureMessage` or `captureException` call, depending on the log level. It then re-triggers to preserve default native behavior:
+This integration captures all `Console API` calls and redirects them to Sentry using the SDK's `captureMessage` or `captureException` call, depending on the log level. It then re-triggers to preserve default native behavior.
+
+<Alert level="warning" title="Note">
+
+By default React [logs all errors to the console](https://github.com/facebook/react/blob/493f72b0a7111b601c16b8ad8bc2649d82c184a0/packages/react-reconciler/src/ReactFiberErrorLogger.js#L85), even if you are using an React error boundary. This means that Sentry will capture the error through the `CaptureConsole` integration and not through the error boundary.
+
```

Comment:
reading through the [issue](https://github.com/getsentry/sentry-javascript/issues/8689), should we add a sentence around recommending to disable the `error` log level of CaptureConsole when using react?

Reply:
good idea, will do